common/src/main/java/whocraft/tardis_refined/registry/ItemRegistry.java

```diff
@@ -2,18 +2,31 @@
 
 import dev.architectury.injectables.annotations.ExpectPlatform;
 import net.minecraft.core.Registry;
+import net.minecraft.core.registries.Registries;
+import net.minecraft.world.item.BlockItem;
 import net.minecraft.world.item.CreativeModeTab;
 import net.minecraft.world.item.Item;
+import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.block.SoundType;
 import net.minecraft.world.level.block.state.BlockBehaviour;
 import net.minecraft.world.level.material.Material;
 import whocraft.tardis_refined.TardisRefined;
+import whocraft.tardis_refined.client.TardisClientData;
 import whocraft.tardis_refined.common.block.shell.RootedShellBlock;
 import whocraft.tardis_refined.common.block.shell.ShellBaseBlock;
 import whocraft.tardis_refined.common.items.KeyItem;
+import whocraft.tardis_refined.common.util.Platform;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
 
 public class ItemRegistry {
 
+    public static final ArrayList<Item> CREATIVE_ITEMS = new ArrayList<>();
+    // Forge can't use registry suppliers during startup
+    public static final ArrayList<RegistrySupplier<?>> FORGE_CREATIVE_ITEMS = new ArrayList<>();
```

Comment:
This workaround looks a bit messy, but not sure if this is affected by a possible limitation of RegistrySupplier. @Suff99 any thoughts on this?